.happy/terraform/modules/sfn/main.tf

```diff
@@ -17,6 +17,9 @@ resource "aws_sfn_state_machine" "state_machine" {
           "JobDefinition": "${var.job_definition_arn}",
           "JobName": "download-validate",
           "JobQueue": "${var.job_queue_arn}",
+          "RetryStrategy": {
+            "Attempts": ${var.max_attempts}
```

Comment:
Is this the total number of attempts? Or total number of retries, i.e., `total number of attempts` - 1